samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_2/Adapter/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using ServiceControl.TransportAdapter;
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.ServiceControl.RabbitMQAdapter.Adapter";
+        #region AdapterTransport
+
+        var transportAdapter =
+            new TransportAdapterConfig<RabbitMQTransport, RabbitMQTransport>("ServiceControl.RabbitMQ.Adapter");
+
+        transportAdapter.CustomizeServiceControlTransport(
+            customization: transport =>
+            {
+                transport.ConnectionString("host=localhost");
+                var delayedDelivery = transport.DelayedDelivery();
+                delayedDelivery.DisableTimeoutManager();
```

Comment:
Needs to be removed based on the [TC error](https://builds.particular.net/viewLog.html?buildId=272957&buildTypeId=DocsParticularNet_Snippets_1Build&tab=buildLog&_focus=355#_state=121).